tensorbay/geometry/tests/test_polyline.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+import numpy as np
+import pytest
+
+from .. import Box2D, Polyline2D, Vector2D
+
+_POLYLINE_INFO_1 = [
+    {
+        "vector": np.array([0, 1], dtype=np.float32),
+        "last_time": 0,
+        "time": 0.5,
+        "point": np.array([1, 0], dtype=np.float32),
+        "index": 0,
+    },
+    {
+        "vector": np.array([1, 0], dtype=np.float32),
+        "last_time": 0.5,
+        "time": 1.0,
+        "point": np.array([1, 2], dtype=np.float32),
+        "index": 1,
+    },
+]
+_POLYLINE_INFO_2 = [
+    {
+        "vector": np.array([-9, 12], dtype=np.float32),
+        "last_time": 0,
+        "time": 0.75,
+        "point": np.array([10, -9], dtype=np.float32),
+        "index": 0,
+    },
+    {
+        "vector": np.array([3, 4], dtype=np.float32),
+        "last_time": 0.75,
+        "time": 1.0,
+        "point": np.array([1, 3], dtype=np.float32),
+        "index": 1,
+    },
+]
+_POLYLINE_DATA = [{"x": 1, "y": 1}, {"x": 1, "y": 2}, {"x": 2, "y": 2}]
+
+
+class TestPolyline2D:
+    def test_init(self):
+        sequence = [[1, 1], [1, 2], [2, 2]]
+        assert Polyline2D() == Polyline2D([])
+        assert Polyline2D(sequence) == Polyline2D([Vector2D(1, 1), Vector2D(1, 2), Vector2D(2, 2)])
+
+    def test_eq(self):
+        polyline_1 = Polyline2D([[1, 2], [2, 3], [2, 2]])
+        polyline_2 = Polyline2D([[1, 2], [2, 3], [2, 2]])
+        polyline_3 = Polyline2D([[1, 2], [3, 4], [2, 2]])
+        assert (polyline_1 == polyline_2) == True
+        assert (polyline_1 == polyline_3) == False
+
+    def test_distance(self):
+        sequence_1 = [1, 2]
+        sequence_2 = [4, 6]
+        assert Polyline2D._distance(sequence_1, sequence_2) == 5.0
+
+    def test_get_polyline_info(self):
+        sequence = [[1, 1], [1, 2], [2, 2]]
+        Polyline2D_info = Polyline2D._get_polyline_info(sequence)[0]
```

Comment:
It is necessary to test all the elements in the derived polyline2D info?
And usually, we use lowercase in variable names.